superset/cachekeys/api.py

```diff
@@ -40,9 +46,74 @@ class CacheRestApi(BaseSupersetModelRestApi):
     class_permission_name = "CacheRestApi"
     include_route_methods = {
         "invalidate",
+        "warm_up_cache",
     }
 
-    openapi_spec_component_schemas = (CacheInvalidationRequestSchema,)
+    openapi_spec_component_schemas = (
+        CacheInvalidationRequestSchema,
+        CacheWarmUpRequestSchema,
+        CacheWarmUpResponseSchema,
+    )
+
+    @expose("/warm_up_cache", methods=["PUT"])
```

Comment:
What do you think about calling this just `warm_up` ?

Reply:
keeping as `warm_up_cache` since I've moved the endpoint under `api/v1/charts/`